app/browser/tabs.js

```diff
@@ -699,22 +698,6 @@ const api = {
     })
   },
 
-  executeScriptInBackground: (script, cb) => {
```

Comment:
I think this is being used in a PR in progress

Reply:
Yes this is used in this PR #10325